docs/bugs.json

```
@@ -1,4 +1,15 @@
 [
+    {
+        "uid": "SOL-2022-1",
+        "name": "AbiEncodeCallLiteralAsFixedBytesBug",
+        "summary": "Literals used in place of a fixed bytes argument in ``abi.encodeCall`` were encoded incorrectly",
```

```suggestion
        "summary": "Literal numbers used for a fixed length bytes parameter in ``abi.encodeCall`` were encoded incorrectly.",
```
(I assume "123" works correctly)

---

Kept it at "literals"... decimal literals and hex literals of incorrect length won't convert, i.e. fail in the type checker... string literals, however, are even more messed up and encoded as ``string memory``... which can also happen for any string shorter than 32 bytes and any larger fixed bytes type (i.e. ``"ab"`` will pass type checking as ``bytes32`` and encode a tail pointer to a memory string...)